package.json

```diff
@@ -18,15 +18,15 @@
     "test": "tsdx test",
     "test:watch": "tsdx test --watch",
     "test:e2e": "tsdx test --config jest-e2e-config.json",
-    "lint": "tsdx lint --fix",
-    "prepare": "tsdx build",
+    "lint": "eslint --fix src",
```

Comment:
Should we be running prettier as well? Or just eslint?

Reply:
i added `lint-check`